pyproject.toml

```diff
@@ -93,6 +97,7 @@ doc = [
 
 [project.scripts]
 get-modflow = "flopy.utils.get_modflow:cli_main"
+gen-classes = "flopy.mf6.utils.generate_classes:cli_main"
```

Comment:
`gen-classes` as shortcut for the unwieldy `python -m flopy.mf6.utils.generate_classes`

Reply:
On second thought, I will take this out &mdash; probably worth thinking about what to name this (and whether to factor developer commands differently) in flopy4, and avoid people getting used to something in the meantime.